zio-http/src/main/scala/zhttp/service/Client.scala

```diff
@@ -175,12 +175,16 @@ object Client {
     val url: URL       = endpoint._2
   }
 
-  final case class ClientResponse(status: Status, headers: List[Header], private val buffer: ByteBuf)
+  final case class ClientResponse(status: Status, headers: List[Header], private val unsafeByteBuf: ByteBuf)
       extends HeaderExtension[ClientResponse] { self =>
-    def getBodyAsString: Task[String] = Task(buffer.toString(self.getCharset))
+
+    def getBodyAsString: Task[String] = Task(unsafeByteBuf.toString(self.getCharset))
+
+    def getRawBody: Task[Chunk[Byte]] = Task(Chunk.fromArray(ByteBufUtil.getBytes(unsafeByteBuf)))
```

Comment:
Let's maintain consistent naming conventions. In `Request` this method is called as `getBody` and there is another method for `getBodyAsByteBuf` which is private and returns a ByteBuf.